src/Compilers/Core/MSBuildTask/ManagedCompiler.cs

```diff
@@ -211,6 +224,12 @@ public ITaskItem[] ResponseFiles
             get { return (ITaskItem[])_store[nameof(ResponseFiles)]; }
         }
 
+        public bool SkipCompilerExecution
```

Comment:
What does it mean to set this property but not set the other one? I can imagine some epic practical jokes of setting this property somewhere and watching people wonder why their builds are broken.